lib/internal/process/esm_loader.js

```diff
@@ -65,18 +63,18 @@ async function initializeLoader() {
   // A separate loader instance is necessary to avoid cross-contamination
   // between internal Node.js and userland. For example, a module with internal
   // state (such as a counter) should be independent.
-  const internalEsmLoader = new ESMLoader();
+  const internalEsmLoader = new ESMLoader({ isInternal: true });
```

Comment:
Re above, I think this shouldn’t be necessary.

```suggestion
  const internalEsmLoader = new ESMLoader();
```